test/autorest/generated/azurespecialsgroup/xmsclientrequestid.go

```diff
@@ -63,7 +64,15 @@ func (client *xmsClientRequestIdoperations) getHandleResponse(resp *azcore.Respo
 
 // getHandleError handles the Get error response.
 func (client *xmsClientRequestIdoperations) getHandleError(resp *azcore.Response) error {
-	return errors.New(resp.Status)
+	msg := resp.Status
```

Comment:
I think we should omit this if there's a response body.

Reply:
Yes, I agree. Will update

Reply:
Done